bin/localicious-install

```diff
@@ -24,17 +24,17 @@ loadYaml(localiciousConfig).onSuccess(localiciousConfig => {
       })
       .flatMap(configuration => {
         const { path, options } = configuration;
-        
+
         return validateAndRender(
           path,
           outputPath,
           options.languages,
           options.platforms
-        )
-          
-      }).onSuccess(() => {
+        );
+      })
+      .onSuccess(() => {
         console.log(
-          `✅ Your Localicipe was downloaded and rendered successfully for platforms: ${options.platforms} and languages: ${options.languages}.`
+          `✅ Your Localicipe was downloaded and rendered successfully.`
```

Comment:
As discussed offline: let's change it to`Your Localicipe was rendered successfully` since that's all the user is interested in + the Localicipe can also be loaded locally